common/vendor-src.js

```diff
@@ -15,7 +15,8 @@ if (typeof platform_require !== "undefined") {
 const vendorModules = {
   "redux": require("redux"),
   "redux-thunk": require("redux-thunk"),
-  "reselect": require("reselect")
+  "reselect": require("reselect"),
+  "fancy-dedupe": require("fancy-dedupe")
```

Comment:
@k88hudson any reason not to do this? Tldr need deduping to work in *Feed.js file. Full discussion here: https://github.com/mozilla/activity-stream/issues/2622#issuecomment-307789036